pages/windows/ospp.vbs.md

```diff
@@ -0,0 +1,29 @@
+# ospp.vbs
+
+> Install, activate, and manage volume licensed versions of Office products.
```

Comment:
```suggestion
> Install, activate, and manage volume licensed versions of Microsoft Office products.
```

If this program works exclusively with Microsoft Office, then I would mention the name Microsoft at least once in the description.